gp-conditional-pricing/gpcp-fuzzy-comparison.php

```diff
@@ -0,0 +1,190 @@
+<?php
+/**
+ * Gravity Perks // GP Conditional Pricing // Fuzzy Comparison for operator "is"
+ * http://gravitywiz.com/documentation/gravity-forms-conditional-pricing/
+ *
+ * This snippet adds fuzzy comparison for "is" operator with the GPCP rules.
+ *
+ */
+class GPCP_Fuzzy_Match {
```

Comment:
Let's rename this to `GW_Fuzzy_Match` and move it to the `gravity-forms` folder since it applies to all GF conditional logic. 🙌